aeron-samples/src/main/java/io/aeron/samples/AeronStat.java

```diff
@@ -65,6 +66,11 @@
      */
     private static final String WATCH = "watch";
 
+    /**
+     * Whether to output in Influx format
+     */
+    private static final String TELEGRAF = "telegraf";
```

Comment:
It would be better to have an output `format` option that would default to `basic` and could be `telegraf`.